services/workflow_orchestrator.py

```
@@ -776,6 +776,33 @@ async def _checkpoint(self, run: WorkflowRun) -> None:
+            risk_reason = ""
+            if plan is not None and plan.requires_risky_review:
+                risk_reason = "Plan touches a sensitive/risky path (requires_risky_review=true)."
+
+            NotificationDispatcher().send_approval_gate(
+                run_id=run.run_id,
+                company_id=run.company_id,
+                goal=goal,
+                plan_steps=steps,
+                risk_reason=risk_reason,
+            )
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)